### PR TITLE
[TACHYON-1257] Fix bug FileSystemMaster causing FileSystemMasterIntegrationTest flaky

### PR DESCRIPTION
**Pull request by apc999**
https://tachyon.atlassian.net/browse/TACHYON-1257

### CODE REVIEW

File: servers/src/main/java/tachyon/master/file/FileSystemMaster.java [hunk 1]
```diff
@@ -914,7 +914,8 @@ private boolean isInMemory(BlockInfo blockInfo) {
       try {
         CreatePathOptions createPathOptions =
             new CreatePathOptions.Builder(MasterContext.getConf()).setDirectory(true)
-                .setPersisted(options.isPersisted()).setRecursive(options.isRecursive()).build();
+                .setPersisted(options.isPersisted()).setRecursive(options.isRecursive())
+                .setOperationTimeMs(options.getOperationTimeMs()).build();;
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/file/FileSystemMaster.java]: double semicolon

**Comment by gpang** [comment]
I had a minor style comment. Thanks for this fix!

**Comment by calvinjia** [comment]
Aside from the style comment, LGTM

**Comment by jsimsa** [comment]
ditto

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by gpang** [comment]
LGTM

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**